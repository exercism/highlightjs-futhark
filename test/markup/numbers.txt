1 200_000 0x123 0b0101 4i8 6i16 7i32 8i64 11u8 12u16 13u32 14u64 .2 0.23 1.0f16 23.24f32 123f64