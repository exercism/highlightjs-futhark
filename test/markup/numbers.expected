<span class="hljs-number">1</span> <span class="hljs-number">200_000</span> <span class="hljs-number">0x123</span> <span class="hljs-number">0b0101</span> <span class="hljs-number">4i8</span> <span class="hljs-number">6i16</span> <span class="hljs-number">7i32</span> <span class="hljs-number">8i64</span> <span class="hljs-number">11u8</span> <span class="hljs-number">12u16</span> <span class="hljs-number">13u32</span> <span class="hljs-number">14u64</span> <span class="hljs-number">.2</span> <span class="hljs-number">0.23</span> <span class="hljs-number">1.0f16</span> <span class="hljs-number">23.24f32</span> <span class="hljs-number">123f64</span>